Sources/TensorFlow/Core/Runtime.swift

```diff
@@ -934,65 +934,33 @@ public func _tffunc<State: _TensorArrayProtocolEnhanced, Data: TensorGroup>(
     }
 }
 
+internal func _trace<In: TensorGroup, Out: TensorGroup>(_ fn: (In) -> Out) -> TFFunction {
+    let useLazyTensor = _RuntimeConfig.useLazyTensor
```

Comment:
This seems not thread-safe.

Reply:
Ah.. you are right. Hold the review on this.

Reply:
This should be resolved now.